Realm/RealmSwift/List.swift

```diff
@@ -72,28 +120,57 @@ public final class List<T: Object>: ListBase, SequenceType {
         }
     }
 
+    /// Returns the first object in the list, or `nil` if empty.
     public var first: T? { return _rlmArray.firstObject() as T? }
 
+    /// Returns the last object in the list, or `nil` if empty.
     public var last: T? { return _rlmArray.lastObject() as T? }
 
     // MARK: Subarray Retrieval
 
+    /**
+    Filters the List to the objects that match the given predicate.
+
+    :param: predicateFormat The predicate format string which can accept variable arguments.
+
+    :returns: Results containing objects that match the given predicate.
+    */
     public func filter(predicateFormat: String, _ args: CVarArgType...) -> Results<T> {
         return Results<T>(_rlmArray.objectsWithPredicate(NSPredicate(format: predicateFormat, arguments: getVaList(args))))
     }
 
+    /**
+    Filters the List to the objects that match the given predicate.
+
+    :param: predicateFormat The predicate to filter the objects.
+
+    :returns: Results containing objects that match the given predicate.
+    */
     public func filter(predicate: NSPredicate) -> Results<T> {
         return Results<T>(_rlmArray.objectsWithPredicate(predicate))
     }
 
     // MARK: Sorting
 
+    /**
+    Returns a sorted version of the list.
+
+    :param: property  The property name to sort by.
+    :param: ascending The direction to sort by.
+
+    :returns: Results containting the objects in the list sorted by the given property.
+    */
     public func sorted(property: String, ascending: Bool = true) -> Results<T> {
         return Results<T>(_rlmArray.sortedResultsUsingProperty(property, ascending: ascending))
     }
 
-    // MARK: Sequence Support
+    // MARK: Sequence support
 
+    /**
+    Returns a `GeneratorOf<T>` that yields successive elements in the list.
+
+    :returns: Returns a `GeneratorOf<T>` that yields successive elements in the list.
```

Comment:
omit the "Returns":

`:returns: A `GeneratorOf<T>` that yields successive elements in the list.`